src/devices/ft004b.c

```diff
@@ -0,0 +1,67 @@
+#include "rtl_433.h"
```

Comment:
Hi, we need a license header and it would be nice with a brief signal description. Other then that the patch is ready for merging.

Reply:
Thanks, should be fixed now!